<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcSearchWidget</class>
 <widget class="QWidget" name="MusicLrcSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>575</width>
     <height>395</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌词搜索</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="songNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>62</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lrcSearchButton">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>60</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="songSearchEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>62</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicLrcSearchTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>155</y>
     <width>535</width>
     <height>210</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>375</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>375</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>575</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <widget class="QLabel" name="functionTopLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>535</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
   <property name="text">
    <string> 暂无所搜到的歌词</string>
   </property>
  </widget>
  <widget class="QLabel" name="functionBottomLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>123</y>
     <width>535</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(240,240,240)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_song">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌曲名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_art">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold</string>
   </property>
   <property name="text">
    <string>艺术家</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold</string>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="label_checkBox">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>132</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QCheckBox{ background-color:transparent; } 
 QCheckBox::indicator::unchecked{ image: url(:/control/checkbox_unchecked); }QCheckBox::checked { color: rgb(230,115,0); } 
QCheckBox::indicator:checked { image: url(:/control/checkbox_checked); };</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lrcSearchDownload">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>375</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>songNameLabel</zorder>
  <zorder>lrcSearchButton</zorder>
  <zorder>songSearchEdit</zorder>
  <zorder>tableWidget</zorder>
  <zorder>stateLabel</zorder>
  <zorder>closeButton</zorder>
  <zorder>functionBottomLabel</zorder>
  <zorder>label_song</zorder>
  <zorder>label_art</zorder>
  <zorder>label_time</zorder>
  <zorder>functionTopLabel</zorder>
  <zorder>label_checkBox</zorder>
  <zorder>lrcSearchDownload</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicLrcSearchTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musiclrcsearchtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
